src/config.rs

```diff
@@ -384,37 +357,20 @@ static:
         assert_static_endpoints(
             &config.source,
             vec![
-                EndPoint::with_metadata(
+                Endpoint::with_metadata(
                     "127.0.0.1:26000".parse().unwrap(),
-                    Some(
-                        serde_yaml::to_value(
-                            vec![(
-                                "quilkin.dev",
-                                vec![("tokens", vec!["MXg3aWp5Ng==", "OGdqM3YyaQ=="])]
-                                    .into_iter()
-                                    .collect::<HashMap<_, _>>(),
-                            )]
+                    EndpointMetadata {
+                        tokens: vec!["1x7ijy6", "8gj3v2i"]
```

Comment:
how come these aren't base64 anymore, what's the change?

Reply:
Oh right whoops, I forgot I meant to mention that. The change is that the token metadata's internal representation is now the binary of the token, rather than being the binary of the base64 of the token, and the token is now only converted to and from base64 when converting it to or from a different representation such as YAML, JSON, Protobuf.